pyproject.toml

```diff
@@ -118,7 +117,3 @@ juju = "^3.6"
 tenacity = "^9.0.0"
 pytest = "^8.3.3"
 pytest-operator = ">0.30"
```

Comment:
suggestion: include all dependencies needed by integration tests so we do not have to install charm dependencies for integration tests, similar to pvcviewer